darts/models/forecasting/arima.py

```diff
@@ -45,20 +53,29 @@ def __init__(
 
         Parameters
         ----------
-        p : int
-            Order (number of time lags) of the autoregressive model (AR).
+        p : int | Sequence[int]
+            Order (number of time lags) of the autoregressive model (AR). A
+            sequence of integers can be given specifying exactly which lag
+            orders are included.
         d : int
             The order of differentiation; i.e., the number of times the data
             have had past values subtracted (I).
-        q : int
-            The size of the moving average window (MA).
-        seasonal_order: Tuple[int, int, int, int]
+        q : int | Sequence[int]
+            The size of the moving average window (MA). A sequence of integers
+            can be given specifying exactly which lag orders are included.
+        seasonal_order: Tuple[int | Sequence[int], int, int | Sequence[int], int]
             The (P,D,Q,s) order of the seasonal component for the AR parameters,
-            differences, MA parameters and periodicity.
+            differences, MA parameters and periodicity. D and s are always integers,
+            while P and Q may either be integers or sequence of positive integers
+            specifying exactly which lag orders are included.
         trend: str
-            Parameter controlling the deterministic trend. 'n' indicates no trend,
-            'c' a constant term, 't' linear trend in time, and 'ct' includes both.
-            Default is 'c' for models without integration, and no trend for models with integration.
+        trend: Literal['n', 'c', 't', 'ct'] | Sequence[int], optional
+            Parameter controlling the deterministic trend. Can be specified as
+            a string where 'c' indicates a constant term, 't' indicates a
+            linear trend in time, and 'ct' includes both. Can also be specified
+            as an sequence defining a polynomial, as in `numpy.poly1d`, where
+            `[1,1,0,1]` would denote :math:`a + bt + ct^3`. Default is 'c' for
+            models without integration, and no trend for models with integration.
```

Comment:
you can remove the left over line from above (trend: str)


```suggestion
        trend: Literal['n', 'c', 't', 'ct'] | Sequence[int], optional
            Parameter controlling the deterministic trend. Either a string or sequence of integers.
            If a string, can be 'n' for no trend, 'c' for a constant term, 't' for a linear trend in time,
            and 'ct' for a constant term and linear trend.
            If a sequence of integers, defines a polynomial according to `numpy.poly1d` [1]_. E.g., `[1,1,0,1]` would
            translate to :math:`a + bt + ct^3`.
            Default is 'c' for models without integration, and 'n' for models with integration.
```

For the reference (`[1]_`) you have to add the following below the parameter descriptions (above the example)

```
        References
        ----------
        .. [1] https://numpy.org/doc/stable/reference/generated/numpy.poly1d.html
```

Reply:
Will the reference [1] in ARIMA's doc collide with the file docstring?